### PR TITLE
Added appveyor.yml

### PR DESCRIPTION
**Pull request by AnthonySteele**
I added an appveyor.yml file
~~.It's probaby wrong somehow as I don't really know what I'm doing with those~~ It seems to be working on appveyor now.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+version: 0.0.{build}
```
Comment (petemounce) [inline appveyor.yml]: I think the only thing needing a change is this - it should be `1.0.0.{build}` I think. The current release is `0.76.0` - let's go 1.0 with the release that has CI.

**Comment by petemounce** [comment]
I've set up https://ci.appveyor.com/project/justeattech/justbehave

**Comment by AnthonySteele** [comment]
It seems to be working now!

**Comment by petemounce** [comment]
Beautiful - go grab a badge from http://shields.io/ for our readme, and away we go!

**Comment by AnthonySteele** [comment]
How does that badge work - looks like a static image not linked to actual appveyor status?

**Comment by petemounce** [comment]
@AnthonySteele I think it's actually a RESTful endpoint that, under the hood, will ask appveyor for the latest status.

**Pull request merged**